### ISSUE TITLE
Add `profile.release` to enable link time optimizations

### ISSUE THREAD

**Issue by rawhuul** [squidowl/halloy#361]
Hi, the default `profile.release` for `Cargo.toml` is:

**Comment by tarkah** [comment]
Hey @rawhuul thanks for the suggestion! I'd like to see us create a special profile specifically called `packaging` that can use these optimizations.

I use `release` profile a lot while developing and LTO will really kill compile times there, so I think we should only enable when "packaging" the app for release

**Comment by rawhuul** [comment]
Yup, you are right. This can be achieved easily, this feature is stable from `rustc v1.57`

```toml
[profile.packaging]
inherits = "release"
lto = true
```

**Comment by hashcatHitman** [comment]
Is this still something that is wanted? I don't see any clear reason #367 was closed, and I'd be happy to make a quick PR if this is still something we want.

**Comment by auronandace** [comment]
Does #325 have an impact here?
